src/components/StatusFilter.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useRef } from 'react'
+import FilterListIcon from '@material-ui/icons/FilterList'
+import Popover from '@material-ui/core/Popover'
+import Checkbox from '@material-ui/core/Checkbox'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import IconButton from '@material-ui/core/IconButton'
+
+const StatusFilter = ({ status, setStatus }) => {
+  const [open, setOpen] = useState(false)
+  const buttonEl = useRef(null)
```

Comment:
what useRef hook do? and buttonEl dost really tell me what it does

Reply:
A React hook that can store elements as references. In the Popover, we use this reference to anchor it to the filter button, so it doesn't pop out somewhere else random.